src/test/groovy/edu/ie3/datamodel/io/sink/CsvFileSinkTest.groovy

```diff
@@ -140,91 +113,6 @@ class CsvFileSinkTest extends Specification implements TimeSeriesTestData {
-    WecResult wecResult = new WecResult(uuid, TimeUtil.withDefaults.toZonedDateTime("2020-01-30 17:26:44"), inputModel, p, q)
-    EvcsResult evcsResult = new EvcsResult(uuid, TimeUtil.withDefaults.toZonedDateTime("2020-01-30 17:26:44"), inputModel, p, q)
-    EmResult emResult = new EmResult(uuid, TimeUtil.withDefaults.toZonedDateTime("2020-01-30 17:26:44"), inputModel, p, q)
-
-    Quantity<Power> pRef = Quantities.getQuantity(5.1, StandardUnits.ACTIVE_POWER_RESULT)
-    Quantity<Power> pMin = Quantities.getQuantity(-6, StandardUnits.ACTIVE_POWER_RESULT)
-    Quantity<Power> pMax = Quantities.getQuantity(6, StandardUnits.ACTIVE_POWER_RESULT)
-    FlexOptionsResult flexOptionsResult = new FlexOptionsResult(uuid, TimeUtil.withDefaults.toZonedDateTime("2020-01-30 17:26:44"), inputModel, pRef, pMin, pMax)
-
-    when:
-    csvFileSink.persistAll([
-      pvResult,
-      wecResult,
-      evcsResult,
-      emResult,
-      flexOptionsResult,
-      GridTestData.transformerCtoG,
-      GridTestData.lineGraphicCtoD,
-      GridTestData.nodeGraphicC,
-      ThermalUnitInputTestData.cylindricStorageInput,
-      ThermalUnitInputTestData.thermalHouseInput,
-      SystemParticipantTestData.evcsInput,
-      SystemParticipantTestData.loadInput,
-      SystemParticipantTestData.emInput
-    ])
-    csvFileSink.shutdown()
-
-    then:
-    new File(testBaseFolderPath).exists()
-    new File(testBaseFolderPath + File.separator + "wec_res.csv").exists()
-    new File(testBaseFolderPath + File.separator + "pv_res.csv").exists()
-    new File(testBaseFolderPath + File.separator + "evcs_res.csv").exists()
-    new File(testBaseFolderPath + File.separator + "em_res.csv").exists()
-    new File(testBaseFolderPath + File.separator + "flex_options_res.csv").exists()
-    new File(testBaseFolderPath + File.separator + "transformer_2_w_type_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "node_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "transformer_2_w_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "operator_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "cylindrical_storage_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "line_graphic_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "line_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "operator_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "node_graphic_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "thermal_bus_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "thermal_house_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "load_input.csv").exists()
-    new File(testBaseFolderPath + File.separator + "em_input.csv").exists()
-
-    !new File(testBaseFolderPath + File.separator + "ev_res.csv").exists()
-  }
-
```

Comment:
We can and should keep the test. We can just remove the `initFiles` parameter and adjust the test name description to `A valid CsvFileSink without 'initFiles' should persist provided elements correctly`